pkgs/applications/networking/instant-messengers/bitlbee-facebook/default.nix

```diff
@@ -12,6 +12,17 @@ stdenv.mkDerivation rec {
     sha256 = "1yjhjhk3jzjip13lq009vlg84lm2lzwhac5jy0aq3vkcz6rp94rc";
   };
 
+  # TODO: This patch should be included with the next release after v1.2.1
+  #       these lines should be removed when this happens.
+  patches = [
+    (fetchpatch {
+        name = "FB_ORCA_AGENT_version_bump.patch";
+        url = "https://github.com/bitlbee/bitlbee-facebook/commit/49ea312d98b0578b9b2c1ff759e2cfa820a41f4d.patch";
+        sha256 = "1c7ipbjg3560hg90zmg9r599287xfka1f7wxhiqzknikcjrcj8br";
```

Comment:
```suggestion
        sha256 = "0nzyyg8pw4f2jcickcpxq7r2la5wgl7q6iz94lhzybrkhss5753d";
```

Reply:
It's not simply the result of `nix-prefetch-url` with the proper `--name=`? Thanks for the hash, don't have a linux box I want to build on rn.

Reply:
name does not change the hash. fetchpatch normalizes the patch which changes the hash.